go.mod

```diff
@@ -5,20 +5,20 @@ go 1.16
 require (
 	decred.org/cspp v0.3.0
 	github.com/decred/dcrd/addrmgr/v2 v2.0.0
-	github.com/decred/dcrd/blockchain/stake/v4 v4.0.0-20211120053236-81ae286f2347
+	github.com/decred/dcrd/blockchain/stake/v4 v4.0.0
 	github.com/decred/dcrd/blockchain/standalone/v2 v2.0.1-0.20210914212651-723d86274b0d
 	github.com/decred/dcrd/blockchain/v4 v4.0.0-20211120053236-81ae286f2347
-	github.com/decred/dcrd/certgen v1.1.2-0.20210914212651-723d86274b0d
-	github.com/decred/dcrd/chaincfg/chainhash v1.0.4-0.20210914212651-723d86274b0d
+	github.com/decred/dcrd/certgen v1.1.2-0.20211122175041-d1b1fdba5266
```

Comment:
Really no reason to use the non-release versions of `certgen`, `blake256`, and `dcrec`.  They only contain comment typo fixes and other changes that have zero effect which is why I didn't make new releases out of them.

Reply:
I was going to ask about that.  I checked each of those three and what I found is this:
- `certgen` has modernized build tags, but no code changes
- `dcrec` had it's min go version bumped in e3f7f58a6853b69420c076f7959c653b9bfd3703
- `blake256` has a change in b6d8ccdedf78536e1d085b725388def2ba76f6b8 that changed the `digest` `Sum` method from a pointer to a value receiver.  The type is not exported, but the method is.

These all seemed fairly inconsequential, but I wanted to confirm.

Reply:
Yep, that's correct.  I didn't see any reason for the churn given there are no tangible changes.